pkg/tarmak/kubectl/kubectl.go

```diff
@@ -282,9 +282,10 @@ func (k *Kubectl) Kubectl(args []string) error {
 	}
 
 	cmd := exec.Command("kubectl", args...)
-	cmd.Env = []string{
+	cmd.Env = append(
+		os.Environ(),
 		fmt.Sprintf("KUBECONFIG=%s", k.ConfigPath()),
```

Comment:
Just a thought - if the local user already has KUBECONFIG set to something non-empty, which value will take precedence?

Reply:
From golang stdlib:

```
        // Env specifies the environment of the process.
        // Each entry is of the form "key=value".
        // If Env is nil, the new process uses the current process's
        // environment.
        // If Env contains duplicate environment keys, only the last
        // value in the slice for each duplicate key is used.
        Env []string
```

I think this behaviour changed with golang 1.9+